index.html

```diff
@@ -19,14 +19,16 @@
         <nav>
           <div class="logo">Bash</div>
           <div class="openMenu"><img src="./images/Union.svg" alt="hamburger"></div>
-          <ul class="mainMenu">
-            <li class="shieldOff1"><a href="#my-portfolio">Portfolio</a></li>
-            <li class="shieldOff2"><a href="#about-myself">About</a></li>
-            <li class="shieldOff3"><a href="#contact-me">Contact</a></li>
-            <li class="closeMenu"><img src="./images/Close_Icon.svg" alt="closeMenu"></li>
-          </ul>
         </nav>
       </header>
+      <div class="desktop-header">
+        <div class="Dlogo">Bash</div>
+        <ul class="DMenu">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
+      </div>
```

Comment:
Kindly `increase the spacing` between menu links and decrease `font-weight` to match the Figma design
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/162988824-a836320f-d36f-47e0-8479-2a18249dc950.png) | ![image](https://user-images.githubusercontent.com/24703171/162988912-87a8b731-be99-4a84-9336-86ef1ea8bfed.png) |